queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -208,16 +208,29 @@ def reduce(self):
             script_out = os.path.join(log_dir, log_and_err_name + "Script.out")
             mantid_log = os.path.join(log_dir, log_and_err_name + "Mantid.log")
 
-            # strip the temp path off the front of the temp directory to get the final archives
-            # directory.
-            final_result_dir = reduce_result_dir[len(MISC["temp_root_directory"]):]
-            final_log_dir = log_dir[len(MISC["temp_root_directory"]):]
-
-            final_result_dir = self._new_reduction_data_path(final_result_dir)
-            final_log_dir = append_path(final_result_dir, ['reduction_log'])
-
-            logger.info('Final Result Directory = %s', final_result_dir)
-            logger.info('Final Log Directory = %s', final_log_dir)
+            def result_directory():
+                """
+                create and return final result directory, stripping temporary path off the front
+                of temporary directories
+                :return (str) final result directory path
+                """
+                final_result_directory = reduce_result_dir[len(MISC["temp_root_directory"]):]
+                final_result_directory = self._new_reduction_data_path(final_result_directory)
+                logger.info(f"Final Result Directory = {final_result_dir}")
```

Comment:
Pylint (rightly) points out we should not use f strings for logging, since we force the interpreter to eval this now. Instead you should do

`logger.info("Final Result Directory = %s", final_result_dir)`

Then the logger can avoid the operation if were not using info and below. This keeps it consistent if we also add debug too later on